test/pages/OneRepayment.scala

```diff
@@ -42,8 +44,8 @@ object OneRepayment extends CommonPage {
     readTitle shouldBe "Vat Repayment Tracker"
     readMainMessage shouldBe "We are processing your VAT repayment"
     readAmount shouldBe "£5.56"
-    readRepayDate shouldBe "2018-05-12"
-    readReceivedDate shouldBe "2018-04-12"
+    readRepayDate shouldBe LocalDate.now().plusDays(30).toString
```

Comment:
here we can hardcode expdected date because there was frozen clock in ItSpec

Reply:
Done